### PR TITLE
Small refactors

### PR DESCRIPTION
**Pull request by daltzctr**

### CODE REVIEW

File: source/docs/troubleshooting/canbus-troubleshooting.rst [hunk 1]
```diff
@@ -47,15 +49,15 @@ Approach 1 (best)
 - **Physically connect CAN bus from roboRIO to one device only.  Circumvent your wiring if need be.**
 - Power boot robot/bench setup.
 - Open Phoenix Tuner X and wait for connection (roboRIO may take ~30 seconds to boot)
-- Open the Devices page
+- Open the :guilabel:`Devices`` page
```
Comment (bhall-ctre) [inline source/docs/troubleshooting/canbus-troubleshooting.rst:52]: There's a trailing \` here

File: source/docs/tuner/index.rst [hunk 2]
```diff
@@ -4,12 +4,12 @@ Phoenix Tuner X
 What is Phoenix Tuner X?
 ------------------------
 
+.. note:: The legacy Phoenix Tuner v1 is still available for use and is pre-installed as part of the Phoenix installer.
```
Comment (bhall-ctre) [inline source/docs/tuner/index.rst:7]: Replace "pre-installed" with "installed"

File: source/docs/api-reference/index.rst [hunk 3]
```diff
@@ -3,6 +3,29 @@
 API Reference
 =============
 
+This section highlights the functionality that is exposed as part of the Phoenix Pro API. This includes the base API, simulation and detailed example projects.
```
Comment (bhall-ctre) [inline source/docs/api-reference/index.rst:6]: Add Oxford comma: "base API, simulation, and detailed example projects"

File: source/docs/api-reference/index.rst [hunk 4]
```diff
@@ -3,6 +3,29 @@
...
+
+.. grid:: 1 2 3 3
+   :gutter: 3
+
+   .. grid-item-card:: API Usage
+      :link-type: doc
+      :link: api-usage/index
```
Comment (bhall-ctre) [inline source/docs/api-reference/index.rst:13]: There are a couple of other places where we have these grid cards, such as the API Usage index.rst. Could you add links to those cards as well?

**Review by bhall-ctre — CHANGES_REQUESTED** [review]

File: source/docs/api-reference/index.rst (hunk 3 above)
Comment (daltzctr) [reply source/docs/api-reference/index.rst:6]: I've typically only used Oxford comma in lists of greater than 3, but I'll add one here anyways. Doesn't affect readability either way.

File: source/docs/api-reference/index.rst (hunk 4 above)
Comment (daltzctr) [reply source/docs/api-reference/index.rst:13]: Could you clarify what you mean by **those cards**? Specifically, what is **those** referring to?

Comment (bhall-ctre) [reply source/docs/api-reference/index.rst:13]: Looking over everything again, it looks like it's just the three cards on the API Usage index.rst page. I feel like all three cards (Configs, Control Requests, and Signals) should probably link to their respective pages.

Comment (daltzctr) [reply source/docs/api-reference/index.rst:13]: Got it. Thanks.

**Review by bhall-ctre — APPROVED** [review]

**Pull request merged**